network/hnswrapper/hnsv2wrapperwithtimeout.go

```diff
@@ -1,12 +1,28 @@
+// Copyright 2022 Microsoft. All rights reserved.
+// MIT License
```

Comment:
we have a global LICENSE in the root of the repo, we do not need license headers unless this file should fall under a different license.

Reply:
removed